test/components/views/rooms/RoomList-test.js

```diff
@@ -296,6 +296,11 @@ describe('RoomList', () => {
             GroupStore._notifyListeners();
 
             await waitForRoomListStoreUpdate();
+
+            // XXX: Even though the store updated, it can take a bit before the update makes
+            // it to the components. This gives it plenty of time to figure out what to do.
+            await (new Promise(resolve => setTimeout(resolve, 500)));
```

Comment:
... also this is a groups test so I don't feel bad for it given it'll soon disappear.